File: MyCallIntent2/app/src/main/java/com/example/mycallintent/MainActivity.java
package com.example.mycallintent;

import androidx.appcompat.app.AppCompatActivity;

import android.content.ComponentName;
import android.content.Intent;
import android.net.Uri;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Button button = findViewById(R.id.button);
        button.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(Intent.ACTION_VIEW, Uri.parse("tel:010-1000-1000"));
                startActivity(intent);

                /*
                Intent intent = new Intent();
                ComponentName name = new ComponentName("com.example.myCallIntent2","com.example.myCallIntent2.MenuActivity");
                intent.setComponent(name);
                startActivityForResult(intent,101);
                 */
            }
        });
    }
}

File: Crime/app/src/main/java/com/example/crime/MainActivity.kt
package com.example.crime

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val currentFragment = supportFragmentManager.findFragmentById(R.id.fragement_container)

        if(currentFragment == null){
            val fragment = CrimeFragment()
            supportFragmentManager.beginTransaction().add(R.id.fragement_container, fragment).commit()
        }
    }
}